.circleci/config.yml

```diff
@@ -0,0 +1,44 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    working_directory: ~/ezcater
+    docker:
+      - image: circleci/ruby:2.3.3
+    steps:
+      - checkout
+
+      # Restore bundle cache
+      - restore_cache:
+          keys:
+            - ruby-cache-{{ .Branch }}-{{ checksum "ezcater_rubocop.gemspec" }}
```

Comment:
In general I think you want both the Gemfile and the gemspec here. We can leave off the Gemfile for now, but there are cases, for example if you need to point to github for a particularly dependency, where you need to do that in the Gemfile. Or use another tool like [appraisal](https://github.com/thoughtbot/appraisal)

Reply:
ah - ok - i will add back in for completeness. Sorry about that!